.github/workflows/smoke-checks.yml

```diff
@@ -88,6 +88,49 @@ jobs:
         MATRIX_CONTEXT: ${{ toJson(matrix) }}
       if: ${{ github.event_name == 'push' && failure() }}
 
+  build-and-test-xcode12-debug:
+    name: Test LLC (Xcode 12, Debug)
```

Comment:
We are going to add 15min+ to our CI for nothing. Tests should not change based on Xcode. We should only build the project not run the full test suite.

Reply:
@tbarbugli handed the PR over to me, and we agreed that you're right. I'm updating the PR so we only build and not test the project.

Reply:
@b-onc Can you please update the job name?

Reply:
In case we are doing 2 lanes, we can name "Build LLC (Xcode 12)" and "Build UI (Xcode 12)"